docs/data/routes/docs/nextjs/troubleshooting/en.md

```diff
@@ -0,0 +1,156 @@
+---
+name: troubleshooting
+routeTemplate: ./data/component-templates/article.yml
+title: Documentation
+---
+
+# Troubleshooting
+
+This section has advice for troubleshooting issues with JSS development.
+
+## Server-side JavaScript errors
+
+If you encounter unexpected JavaScript errors during the `npm install` or `build` steps, especially errors that cannot be reproduced by other team members, check the versions of Node and npm being used. From a terminal, run:
+```
+node -v
+npm -v
+```
+Note that JSS is tested using the Long Term Support (TLS) versions of Node. These are typically one major version behind the latest official Node version.
+
+Note that the Node/npm version used by your CI/Production environments may be different from what is used by your local environment. If a project is not pinned to a specific Node/npm version in `package.json`, it is common for deployment agents to build using either the newest version available or an environment-specific "default" version.
+
+If you need to switch the Node version on your local environment for testing, 3rd party packages like [n](https://github.com/tj/n) or [nvm](https://github.com/nvm-sh/nvm) can help with this task.
+
+System Pre-requisites
+Running Node LTS version (use node -v to check)
+
+## App does not build, or Home Page shows 404
+
+Use the GraphiQL interface to verify that the sitemap query returns the expected list of page routes.
+
+```
+# YOUR_PATH should be the ID of your site root (home page) in lower-case, with dashes removed
+# YOUR_LANGUAGE should be your default language, as defined in package.json
+
+query{
+  search(where: {
+    AND:[
+      {
+      name:"_path",
+      value:"YOUR_PATH"
+      },
+      {
+      name:"_language",
+      value:"YOUR_LANGUAGE"
+      },
+      {
+      name:"_hasLayout",
+      value :"true"
+      }
+    ]
+  })
+  { 
+    total
+    results {
+        url {
+          path
+        }
+      }
+  }
+}
+```
+
+If you do not see expected results, try the following steps:
+
+1. In the **Sitecore Control Panel**, use **Populate Solr Managed Schema** to populate the Solr Schema.
+
+2. In the **Sitecore Control Panel**, use **Indexing Manager** to rebuild indexes.
+
+## Errors regarding SSL certificates
+
+Error examples
+> Error: unable to verify the first certificate
+>
+> Error: UNABLE_TO_VERIFY_LEAF_SIGNATURE
+
+> UnauthorizedError: invalid signature
+
+If you are working with a local Sitecore instance using a privately signed certificate, ensure you've [configured Sitecore CA certificates for Node.js](/docs/temp/node-certificates)"
+
+Note that the steps are different, depending on whether you're using a containerized or on-prem installation of Sitecore.
```

Comment:
Are you referring to the steps for Node certificates? If so, this sentence is redundant, assuming the doc you linked to has a section for each situation.

Reply:
Removed